docs/using-the-jdbc-driver/Telemetry.md

```diff
@@ -0,0 +1,133 @@
+# Monitoring
+
+Monitoring is the ability of gathering data and insights of the execution of an application, then allowing users to inspect the gathered data and determine potential actions to take depending on the data collected.
+
+In version vX.Y.Z of the AWS JDBC Driver, we have introduced a Telemetry feature to the wrapper that enables users to collect and visualize data of the execution of the AWS JDBC Driver, at a global level and at plugin level, meaning that one can now inspect specific performance data for each plugin in the plugin chain.
+
+This enables you to monitor the performance of the driver with their configurations and determine whether that performance meets your expectation on the driver.
+
+## Terminology
+
+The AWS JDBC Driver feature provides telemetry data through two different forms: **Traces** and **Metrics**.
+
+### Traces
+
+Traces give an overview of what is happening in a specific section of the execution of an application. A trace is composed by a hierarchical sequence of segments, each of which contain basic information about the execution (e.g., duration), and whether that section was executed successfully or not.
+
+In the AWS JDBC Driver, initially a trace will be generated for every JDBC call made to the wrapper. Depending on whether the user application has already a trace open, it might be either nested into it or dropped. And then, for each plugin that is part of the wrapper plugin chain, another segment will be created only for the plugin execution, linked to the JDBC call segment.
+
+Traces from the AWS JDBC Driver are submitted to **AWS X-Ray**. [link]
+
+### Metrics
+
+Metrics are numeric data that were measured and collected through the execution of an application. Those metrics can give an insight on how many times some action (e.g., failover) has happened, and if that action succeeded or failed, and other related information.
+
+The AWS JDBC Driver will submit metrics data to **AWS Cloudwatch**.
+
+The list of available metrics available for the AWS JDBC Driver and its plugins is available in the Appendix [link] section of this page.
+
+## Setting up the Amazon OTEL Collector
+
+## Pre Requirements
+
+Before enabling the Telemetry feature, a few setup steps are required to ensure the monitoring data gets properly emitted.
+
+1. In order to visualize the telemetry data in the AWS Console, make sure you have an IAM user or role with permissions to AWS X-Ray and AWS Cloudwatch.
+
+2. Download the AWS OTEL Collector and set it up. The AWS OTEL Collector is responsible from receiving telemetry data from the application using the AWS JDBC Driver and forward it to AWS. Both of those connections happen via HTTP, therefore URLs and ports need to be correctly configured for the collector.
```

Comment:
Is it this one? https://github.com/aws-observability/aws-otel-collector, lets refer to it by `AWS Distro for OpenTelemetry Collector (ADOT Collector)`